### DISCUSSION TITLE
Light/Dark Mode Scheme Not Showing on MKDocs Material

### DISCUSSION THREAD

**Discussion by mangeldev13** [squidfunk/mkdocs-material#5358] [General]
Yesterday I'm successfully add light and dark mode scheme on MkDocs-material When I tried to add auto match with system preferences when I click to auto, the icon is missing. So I delete the code for auto and everything is working fine.

Today, I'm trying to add auto code again, but just like before when I click it, the icon is MISSING.

I try to delete the auto scheme and run mkdocs serve

BUT, NOTHING CHANGES

THE ICON STILL MISSING, what should I do to show up the toggle again? I've rewrite the code and restart my pc

**Comment by squidfunk** [comment]
I'm not sure I understand the problem. It sounds like you're referring to [automatic mode](https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#automatic-light-dark-mode), which is still in [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/). Are you sponsoring and using Insiders? If not, this might be the cause. If that's not what you meant, please provide a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).
